addons/hr_expense/models/hr_expense.py

```diff
@@ -1575,7 +1575,7 @@ def _prepare_bills_vals(self):
         for employee_sudo, expenses_sudo in self.sudo().grouped('employee_id').items():
             multiple_expenses_name = _("Expenses of %(employee)s", employee=employee_sudo.name)
             move_ref = expenses_sudo.name if len(expenses_sudo) == 1 else multiple_expenses_name
-            currency = expenses_sudo.currency_id if len(expenses_sudo.currency_id) == 1 else expenses_sudo.company_currency_id
+            currency = expenses_sudo.company_currency_id
```

Comment:
I would directly set the value in the dict below, no need to use an extra variable